packages/block-library/src/audio/save.js

```diff
@@ -2,13 +2,14 @@
  * WordPress dependencies
  */
 import { RichText } from '@wordpress/block-editor';
+import { getBlockProps } from '@wordpress/blocks';
```

Comment:
I actually wonder about the best place to export this, maybe `useBlockProps.save()`?

Reply:
That's possible too. I don't mind either way. I'll update it :)

Reply:
This looks much nicer actually! :)

Reply:
I'm not sure how I feel about exposing it through `useBlockProps.save()`. It feels a little weird to me, but I also don't outright dislike it. I'm okay with it either way. I suppose one benefit is that it more closely ties it to `useBlockProps`.

Reply:
The reasoning for proposing this is that later we might have `useInnerBlocks.save()` , `useRichText.save()`

Reply:
Yes, I like this pattern. Good idea!